frontend/web/components/Maintenance.tsx

```diff
@@ -1,34 +1,21 @@
 import React from 'react'
 import ConfigProvider from 'common/providers/ConfigProvider'
 
-const HomePage = class extends React.Component {
-  static displayName = 'HomePage'
-
-  constructor(props, context) {
-    super(props, context)
-    this.state = {}
-  }
-
-  render = () => (
+const MaintenancePage: React.FC = () => {
+  return (
     <div className='fullscreen-container maintenance justify-content-center'>
       <div className='col-md-6 mt-5' id='sign-up'>
         <h1>Maintenance</h1>
         We are currently undergoing some scheduled maintenance of the admin
         site, this will not affect your application's feature flags.
-        {
-          <>
-            {' '}
-            Check{' '}
-            <a
-              target='_blank'
-              href='https://x.com/getflagsmith'
-              rel='noreferrer'
-            >
-              @getflagsmith
-            </a>{' '}
-            for updates.
-          </>
-        }
+        <>
+          {' '}
+          Check{' '}
+          <a target='_blank' href='https://x.com/getflagsmith' rel='noreferrer'>
```

Comment:
This is not fundamental part of this PR but we should consider replacing this with https://status.flagsmith.com/ because we're not posting status updates on twitter